Packs/FortiAuthenticator/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### FortiAuthenticator
+- Fixed an issue where the ***fortiauthenticator-update-user*** command for active status (true/false) update
```

Comment:
Did you mean 
Fixed an issue where the ***fortiauthenticator-update-user*** command failed for active status (true/false) update

?

Reply:
Correct.